focused/security/restBasicAuthDBStore/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<persistence version="3.0"
+    xmlns="https://jakarta.ee/xml/ns/persistence"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="
+                https://jakarta.ee/xml/ns/persistence
+                https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd">
```

Comment:
Should be 3.1 for EE10

Reply:
I think there's no persistence 3.1, only an arm 3.1 file.

See https://jakarta.ee/xml/ns/persistence/

Reply:
:o

Reply:
It's slightly confusing indeed.